scripts/zones/Arrapago_Reef/mobs/Medusa.lua

```diff
@@ -19,15 +21,46 @@ function onMobSpawn(mob)
 end;
 
 function onMobEngaged(mob, target)
-    local mobID = mob:getID();
     target:showText(mob, MEDUSA_ENGAGE);
-    SpawnMob(mobID+1, 180):updateEnmity(target);
-    SpawnMob(mobID+2, 180):updateEnmity(target);
-    SpawnMob(mobID+3, 180):updateEnmity(target);
-    SpawnMob(mobID+4, 180):updateEnmity(target);
+    for i = MEDUSA + 1, MEDUSA + 4 do
+        SpawnMob(i):updateEnmity(target);
+    end
+end;
+
+function onMobFight(mob, target)
+    if (mob:getBattleTime() % 15 < 2 and mob:getBattleTime() > 10) then
```

Comment:
im not sure what this is supposed to do

Reply:
if an add is dead will respawn in 10 seconds and update enmity to whoever is tanking